### PR TITLE
feat(cxl-ui): add cxl-dashboard-section component

### PR DESCRIPTION
**Pull request by freudFlintstone**
https://app.clickup.com/t/86ayj5cv3

### CODE REVIEW

**Review by pawelkmpt — CHANGES_REQUESTED** [review]
Squash all commits and change message to

```diff
feat(cxl-ui): add cxl-dashboard-section component

https://app.clickup.com/t/86ayj5cv3
```

**Comment by heshfekry** [comment]
Task linked: [CU-86ayj5cv3 Break out accordion and improve VA](https://app.clickup.com/t/86ayj5cv3)

**Comment by HenerHoop** [comment]
@freudFlintstone You were missing an import from there in the index-core file. I added it. WPS won't work otherwise. Ando also renamed Dashboard section class.

**Comment by HenerHoop** [comment]
@freudFlintstone Fixed height bug and full width description should be limited to a specific width.

Screenshot 2023-11-01 at 12 51 06

File: packages/storybook/cxl-ui/cxl-dashboard/cxl-dashboard-section.stories.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+export default {
+  title: 'CXL UI/cxl-dashboard',
+  component: 'cxl-dashboard-section',
+  parameters: {
+    layout: 'centered',
+  },
+};
+
+const Template = (section) => html`
+  <cxl-dashboard-section image=${section.image}>
```
Comment (HenerHoop) [inline packages/storybook/cxl-ui/cxl-dashboard/cxl-dashboard-section.stories.js:15]: I think we don't need this section image value here.

File: packages/storybook/cxl-ui/cxl-dashboard/cxl-dashboard-sections.stories.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+};
+
+export const CXLDashboardSections = () => html`
+  <div style="margin-top: var(--lumo-space-xl); padding: 0 var(--lumo-space-l);">
+    <div>
+      <h2>Explore all Categories</h2>
+    </div>
+    ${categoryData.map(
+      (cat) => html`
+        <cxl-dashboard-section image=${cat.image}>
```
Comment (HenerHoop) [inline packages/storybook/cxl-ui/cxl-dashboard/cxl-dashboard-sections.stories.js:20]: I think we don't need this category image value here.

**Comment by HenerHoop** [comment]
@freudFlintstone I think we can rebase the master here. Then we'll have the right light card available as well.

**Comment by freudFlintstone** [comment]
You had an outdated version of the branch in your machine, I'll push mine again and you can make changes you need again.

**Comment by pawelkmpt** [comment]
> Squash all commits and change message to
> 
> ```diff
> feat(cxl-ui): add cxl-dashboard-section component
> 
> https://app.clickup.com/t/86ayj5cv3
> ```

@freudFlintstone please read comments carefully. You didn't update commit message

**Comment by HenerHoop** [comment]
> > Squash all commits and change message to
> > ```diff
> > feat(cxl-ui): add cxl-dashboard-section component
> > 
> > https://app.clickup.com/t/86ayj5cv3
> > ```
> 
> @freudFlintstone please read comments carefully. You didn't update commit message

@pawelkmpt It could also have been caused by me because we had a small error there.

**Comment by freudFlintstone** [comment]
commit message edited

**Comment by pawelkmpt** [comment]
> @freudFlintstone Category header has a fixed height issue and full width description should be limited to a specific width.
> 
> Screenshot 2023-11-01 at 12 51 06

Is this @HenerHoop will work on now?

**Review by pawelkmpt — APPROVED** [review]

**Comment by pawelkmpt** [comment]
@freudFlintstone please solve conflicts. They appeared after recent merges

**Review by pawelkmpt — APPROVED** [review]

**Comment by pawelkmpt** [comment]
Task linked: [CU-86ayjehbf Small changes in dashboard (server-side) to accomodate new layout](https://app.clickup.com/t/86ayjehbf)

**Pull request merged**